backend/src/socket/events/addStationChat.js

```diff
@@ -0,0 +1,37 @@
+import * as stationController from '../../controllers/station';
+import * as userController from '../../controllers/user';
+import * as EVENTS from '../../const/actions';
```

Comment:
Why it is "EVENT" but not event?

Reply:
Hi @hynguyen2610 

I am following the code convention from other modules. Every action in backend side called EVENTS and  `uppercase`. You can take a look at other modules in `backend/src/socket/events`.

Reply:
I think that they are constants, so it is better to import as `EVENTS`

Reply:
Alright ;)